src/main/java/JetParser.java

```diff
@@ -5,17 +5,57 @@
 
 import java.io.IOException;
 import java.net.URL;
-
+import java.util.ArrayList;
 public class JetParser {
 
     public static void parsePage() throws IOException {
-        Document doc = Jsoup.connect("https://www.jetphotos.com/registration/ZK-AMY").get();
-        Elements photoWrapper = doc.getElementsByClass("result__section--photo-wrapper");
-        Element imageUrl = photoWrapper.select("result__photo").first();
+        //This ArrayList<jetPhoto> jetPhotos it uses to store jetPhoto objects
+        ArrayList<nz.co.aberg.blip.JetPhoto> jetPhotos = new ArrayList<>();
+        //Retrieving specific resources from the URL
+        Element thumbnailSrc, modelSrc, authorSrc, imageSrc;
+        //Strings are used to store each detail that's already done filter
+        String  thumbnail, mainImage, author, model;
+
+
+        Document doc = Jsoup.connect("https://www.jetphotos.com/registration/ZK-OKQ").get();
```

Comment:
Instead of hard coding the url, we'll pass the url as an argument to the parse method